tests/test_tti.py

```diff
@@ -36,10 +36,10 @@ def test_tti(dimensions, space_order):
 
     model = Model(origin, spacing, dimensions,
                   true_vp, nbpml=nbpml,
-                  epsilon=0.0 * true_vp,
-                  delta=0.0 * true_vp,
-                  theta=0.0 * true_vp,
-                  phi=0.0 * true_vp)
+                  epsilon=None,
```

Comment:
But this is a test, people look at examples, not tests. Anyway, since this is a test, I don't really mind...